qmlexport/CMakeLists.txt

```diff
@@ -20,6 +21,26 @@ if(BUILD_QSKDLL)
         VERSION ${CMAKE_PROJECT_VERSION} SOVERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR} )
 endif()
 
+# packaging
+set(PACKAGE_NAME            QmlExport)
+set(QSKQE_INSTALL_HEADERS   "${CMAKE_INSTALL_INCLUDEDIR}/${target}")
+
 set_target_properties(${target} PROPERTIES FOLDER libs)
+set_target_properties(${target} PROPERTIES EXPORT_NAME ${PACKAGE_NAME})
+
+install(TARGETS ${target} EXPORT ${PACKAGE_NAME}Targets
+    INCLUDES DESTINATION      ${QSKQE_INSTALL_HEADERS}
+    PUBLIC_HEADER DESTINATION ${QSKQE_INSTALL_HEADERS})
+
+export(EXPORT ${PACKAGE_NAME}Targets
+    NAMESPACE ${PACKAGE_NAMESPACE})
+
+install(EXPORT ${PACKAGE_NAME}Targets
+    FILE
+        ${PACKAGE_NAME}Targets.cmake
+    NAMESPACE
+        ${PACKAGE_NAMESPACE}
+    DESTINATION
+        ${PACKAGE_LOCATION})
 
```

Comment:
@uwerat do we want to export this target?

Reply:
The library needs to be linked by the application to access QSkinny from QML. However it is not needed when using the classes from C++ and we do not need to install its header files.

Reply:
I am not sure how to address this. Should the library be installed with the libqskinny.so library at `lib/qskinny` path?

I am not sure how to not include the header files at this time. The only header file being installed is `QskQml.h` at `include/qskqmlexport/` path (this could also be moved to `include/qskinny/` install path if needed, or removed as suggested above).

Reply:
library/header for qskqmlexport can be installed at the same places as library/headers of the core. My statement about not installing the Header was wrong - QskQml.h is needed.

Reply:
Addressed in #362